config/config.xml

```diff
@@ -1,12 +1,19 @@
 <Installer>
     <Name>El-MAVEN</Name>
-    <Version>0.2.4</Version>
+    <Version>x.y.z</Version>
     <Title>El-MAVEN Installer</Title>
     <Publisher>ElucidataInc</Publisher>
-    <ProductUrl>https://elucidatainc.github.io/ElMaven/</ProductUrl>
+    <ProductUrl>https://resources.elucidata.io/elmaven</ProductUrl>
     <InstallerApplicationIcon>maven</InstallerApplicationIcon>
     <InstallerWindowIcon>window.png</InstallerWindowIcon>
     <ControlScript>control_script.qs</ControlScript>
     <WizardStyle>Modern</WizardStyle>
     <TargetDir>@ApplicationsDir@/El-MAVEN</TargetDir>
+    <RemoteRepositories>
+        <Repository>
+            <Url>https://localhost/repository</Url>
```

Comment:
@saifulbkhan 
Why localhost ?

Reply:
Oh this is [updated by the python script](https://github.com/ElucidataInc/elmaven-packaging/pull/4/files#diff-ba585ac1b4ca2bdf42f860bad8c5bf33R10) to a valid S3 URL. The script runs before creating archives, installers, etc.

Reply:
Oops! Missed it